### PR TITLE
[FEAT] Update start date to Nov 4th

### PR DESCRIPTION
**Pull request by eliasSFL**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

# What needs to be tested by the reviewer?

Please describe how this can be tested.

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/lib/flags.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ const featureFlags = {
   CHORE_BOARD: defaultFeatureFlag,
   ONBOARDING_REWARDS: (game: GameState) =>
     game.experiments.includes("ONBOARDING_CHALLENGES"),
-  SEASONAL_TIERS: timeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
+  SEASONAL_TIERS: timeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
```
Comment (eliasSFL) [inline src/lib/flags.ts:42]:
```suggestion
  SEASONAL_TIERS: timeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
```

**Review by c-py — APPROVED** [review]
LGTM! Nice catch!

File: src/lib/flags.ts [hunk 2]
```diff
@@ -48,13 +48,13 @@ const featureFlags = {
...
+  ANIMAL_BUILDINGS: betaTimeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
+  BARLEY: betaTimeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
   GEM_BOOSTS: (game: GameState) => game.experiments.includes("GEM_BOOSTS"),
-  CHICKEN_GARBO: betaTimeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
-  CRAFTING_BOX: betaTimeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
-  FLOWER_BOUNTIES: timeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
-  BEDS: timeBasedFeatureFlag(new Date("2024-11-01T00:00:00Z")),
+  CHICKEN_GARBO: betaTimeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
+  CRAFTING_BOX: betaTimeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
+  FLOWER_BOUNTIES: timeBasedFeatureFlag(new Date("2024-11-04T00:00:00Z")),
```
Comment (c-py) [inline src/lib/flags.ts:56]:
```suggestion
  FLOWER_BOUNTIES: timeBasedFeatureFlag(new Date("2024-11-06T00:00:00Z")),
```

**Review by c-py — CHANGES_REQUESTED** [review]

File: src/lib/flags.ts (hunk 2 above)
Comment (c-py) [reply src/lib/flags.ts:56]: Just spotting one that I think is on the 6th

**Review by c-py — APPROVED** [review]
LGTM!

**Pull request merged**